recipe/meta.yaml

```diff
@@ -1,29 +1,42 @@
 {% set name = "plumbum" %}
-{% set version = "1.8.1" %}
+{% set version = "1.9.0" %}
+{% set tests_to_skip = "" %}
+{% set tests_to_skip = tests_to_skip + "test_cwd or test_mixing_chdir or test_contains or 
+                                        test_path or test_iter_lines_buffer_size" %}
+{% set tests_to_skip = tests_to_skip + " or test_runfile_rich" %}       # [linux]
+{% set tests_to_skip = tests_to_skip + " or test_help_lang" %}          # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_nolang_switches" %}    # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_failure" %}            # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_home" %}               # [win]
```

Comment:
Not you have to change it, but you wouldn't have to skip test_home if you'd patch https://github.com/tomerfiliba/plumbum/blob/master/plumbum/_testtools.py#L16 adding `or sys.platform == "cygwin"`
https://github.com/tomerfiliba/plumbum/blob/v1.9.0/tests/test_env.py#L53

Reply:
Nice catch thanks ! I'll keep the use of "cygwin" in mind for future windows issues.